helm-charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -277,6 +277,14 @@ data:
       github:
         - host: github.com
           token: {{ printf "${%s}" .Values.rhdhOperator.secretRef.github.token }}
+  {{- end }}
+  {{- if and .Values.rhdhOperator.secretRef.gitlab.token (dig "data" .Values.rhdhOperator.secretRef.gitlab.token "" $secret) }}
+      gitlab:
+        - host: {{ printf "${%s}" .Values.rhdhOperator.secretRef.gitlab.host }}
+          token: {{ printf "${%s}" .Values.rhdhOperator.secretRef.gitlab.token }}
```

Comment:
Let's also explicitly check for the host as well, in the render condition.
Something like this?

```
 {{- if and (and .Values.rhdhOperator.secretRef.gitlab.token (dig "data" .Values.rhdhOperator.secretRef.gitlab.token "" $secret) )
             (and .Values.rhdhOperator.secretRef.gitlab.host (dig "data" .Values.rhdhOperator.secretRef.gitlab.host "" $secret  ) ) }}
```